sentry_sdk/integrations/aiohttp.py

```diff
@@ -82,6 +82,8 @@ def setup_once():
                         except HTTPException as e:
                             span.set_http_status(e.status_code)
                             raise
+                        except asyncio.CancelledError:
```

Comment:
Can you invoke `span.set_status("cancelled")` here?

Reply:
Sure thing.